matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/sender/SenderInfo.kt

```diff
@@ -35,3 +36,5 @@ data class SenderInfo(
             else                                      -> "$displayName ($userId)"
         }
 }
+
+fun SenderInfo.toMatrixItem() = MatrixItem.UserItem(userId, displayName, avatarUrl)
```

Comment:
All extensions like this are declared in the file `MatrixItem.kt`.  Maybe move this one there for coherency.

Reply:
Done. 
Thanks !